### ISSUE TITLE
Open diff window on first click of .ipynb file in version control tab

### ISSUE THREAD

**Issue by benlindsay** [microsoft/vscode-jupyter#2466]
# Feature: Notebook Editor, Interactive Window, Python Editor cells

## Description

When clicking on an `.ipynb` file in from the version control tab, when I click the first time, it opens a notebook editor. When I click the second time, it opens the git diff window that I expect. It looks like this was the intentional design in response to microsoft/vscode-python#7905. However, I would appreciate it if the notebook editor wasn't opened at all, and the git diff window opened directly. The notebook editor can take a long time to open, and I have to wait for the whole notebook to load before I can open the git diff window. If it's a large notebook, or if I'm accessing a remote machine, this can take a while. The option to open the notebook editor is already there in the "Open File" button next to the file name in the version control tab.

Thanks for all your hard work on this extension! It's a fantastic tool that I rely on almost every day.

Details:

OS: Mac OS Mojave 10.14.6
VSCode version: 1.41.1
Python Extension version: 2019.11.50794

Microsoft Data Science for VS Code Engineering Team: @rchiodo, @IanMatthewHuff, @DavidKutu, @DonJayamanne, @greazer

**Comment by benlindsay** [comment]
This `"python.dataScience.useNotebookEditor": false` fix used to work fine for me, but at some point it stopped working for me. The behavior when I click on a changed notebook from the Source Control pane is to open 2 notebooks side by side, which do represent the old and new versions, but with no highlighting of any changes. Now it looks like the setting that controls whether to use the notebook editor by default on `.ipynb` files is `"jupyter.useNotebookEditor"`, but that doesn't fix it for me either. There's also another setting: `"notebook.diff.enablePreview"` which also doesn't fix it. 

This is on a remote machine accessed via the remote-ssh extension. I've verified that I've made the settings changes in all 3 possible places, User, Workspace, and Remote:

![image](https://user-images.githubusercontent.com/9361152/110841445-7de1e580-826b-11eb-8cdb-72c5cd3e47b4.png)

![image](https://user-images.githubusercontent.com/9361152/110841511-94883c80-826b-11eb-94d1-73840f7c9457.png)

Any guesses as to why I'm experiencing this and what I can do to fix it?
